tests/commands.test.ts

```diff
@@ -0,0 +1,166 @@
+  });
+
+  octokit = new Octokit({ auth: pat });
+
+  const { data } = await octokit.rest.users.getAuthenticated();
+  username = data.login;
+});
+
+afterAll(async () => {
+  await server.stop();
+});
+
+const waitForNWebhooks = (n: number) =>
+  new Promise<void>((resolve, reject) => {
+    const timer = setTimeout(() => {
+      reject(new Error("timeout"));
+    }, 10000);
+
+    let i = 0;
+    server.probotApp.onAny(() => {
+      i += 1;
+      if (i == n) {
+        clearTimeout(timer);
+        resolve();
+      }
+    });
+  });
+
+const createComment = (issueNumber: number, body: string) => {
+  return octokit.rest.issues.createComment({
+    repo,
+    owner,
+    issue_number: issueNumber,
+    body: body,
+  });
+};
+
+const getLastComment = async (issueNumber: number) => {
+  const { data } = await octokit.rest.issues.listComments({
+    repo,
+    owner,
+    issue_number: issueNumber,
+  });
+  expect(data.length).toBeGreaterThan(0);
+  return data[data.length - 1];
+};
+
+const checkLastComment = async (issueNumber: number, expectedComment: string) => {
+  const lastComment = await getLastComment(issueNumber);
+  expect(lastComment.body).toBe(expectedComment);
+};
+
+describe("commmands test", () => {
+  let issue: Issue;
+
+  beforeAll(async () => {
+    const res = await octokit.rest.issues.create({
+      repo,
+      owner,
+      title: "E2E TEST",
+    });
+    issue = res.data as Issue;
+
+    await waitForNWebhooks(1);
+  }, 10000);
+
+  test("/wallet correct address", async () => {
+    const newWallet = "0x82AcFE58e0a6bE7100874831aBC56Ee13e2149e7";
+    await createComment(issue.number, `/wallet ${newWallet}`);
+    await waitForNWebhooks(2);
+    await checkLastComment(issue.number, `Updated the wallet address for @${username} successfully!\t Your new address: \`${newWallet}\``);
+  }, 10000);
+
+  test("/wallet wrong address", async () => {
+    const newWallet = "0x82AcFE58e0a6bE7100874831aBC56";
+    await createComment(issue.number, `/wallet ${newWallet}`);
+    await waitForNWebhooks(2);
+    await checkLastComment(issue.number, `Please include your wallet or ENS address.\n usage: /wallet 0x0000000000000000000000000000000000000000`);
+  }, 10000);
+
+  test("/multiplier", async () => {
+    await createComment(issue.number, `/multiplier @${username}`);
+    await waitForNWebhooks(2);
+
+    await checkLastComment(issue.number, `Successfully changed the payout multiplier for @${username} to 1. The reason is not provided.`);
+
+    await createComment(issue.number, `/multiplier @${username} 2`);
+    await waitForNWebhooks(2);
+
+    await checkLastComment(
+      issue.number,
+      `Successfully changed the payout multiplier for @${username} to 2. The reason is not provided. This feature is designed to limit the contributor's compensation for any bounty on the current repository due to other compensation structures (i.e. salary.) are you sure you want to use a bounty multiplier above 1?`
```

Comment:
Do you think it's feasible that we export and use the exact same strings from the code base?

Reply:
That would be ideal, but I'm not quite sure how to do it elegantly, because we need to replace username in the string, so we would need to do something like `Successfully changed the payout multiplier for @%username%` and then use `.replace("%username%, username)`